content/install_maintain/installation/installing-external-db-and-queue-cluster.md

```diff
@@ -0,0 +1,215 @@
+ Cloudify Management service cluster nodes configuration.  
+- Keep your RabbitMQ username and password for the later configuration of the Cloudify Management service cluster nodes. 
+- **Note** Reverse DNS lookup must be available in your network for the RabbitMQ nodes, 
+please refer to  [RabbitMQ networking guide - DNS](https://www.rabbitmq.com/networking.html#dns-reverse-dns-lookups)
+ for further explanation.  
+
+
+#### Cloudify Management Service configuration with external services
+
+* In case of an **externally hosted PostgreSQL database** and **externally hosted RabbitMQ** i.e. "bring your own",configure the following settings in `/etc/cloudify/config.yaml`:
+   * **Notice** Some of the keys in the 'postgresql_client' section are relevant only for a few cloud services. Make sure you read the comments
+   provided and follow them.
+
+
+```yaml
+manager:
+    security:
+        ssl_enabled: true
+        admin_password: '<strong admin password for cloudify>'
+
+    cloudify_license_path: '<path to cloudify license file>'
+
+rabbitmq:
+    username: '<username configured for queue management on rabbit>'
+    password: '<strong password configured for queue management on rabbit>'
+
+    # The CA path that was retrieved from the RabbitMQ instance
+    ca_path: '<path to ca certificate>'
+
+    # In case the connection to the RabbitMQ instance uses one IP address,
+    # please specify it as the first cluster node and leave the rest blank
+    cluster_members:
+        <short host name of rabbit server 1>:
+            node_id: <the node`s id> # The node_id can be retrieved by running `cfy_manager node get_id` on the relevant node
+            networks:
+                default: <private ip of rabbit server 1>
+                <other network name>: <address for this node on `other network`>
+        <short host name of rabbit server 2>:
+            node_id: <the node`s id>
+            networks:
+                default: <private ip of rabbit server 2>
+                <other network name>: <address for this node on `other network`>
+        <short host name of rabbit server 3>:
+            node_id: <the node`s id>
+            networks:
+                default: <private ip of rabbit server 3>
+                <other network name>: <address for this node on `other network`>
+
+postgresql_server:
+    # If you are not using a PostgreSQL cluster the 'cluster' section should not be filled.
+    cluster:
+        nodes:
+            <first postgresql instance-name>:
+                ip: <private ip of postgres server 1>
+                node_id: <the node`s id> # The node_id can be retrieved by running `cfy_manager node get_id` on the relevant node
+            <second postgresql instance-name>:
+                ip: <private ip of postgres server 2>
+                node_id: <the node`s id>
+            <third postgresql instance-name>:
+                ip: <private ip of postgres server 3>
+                node_id: <the node`s id>
```

Comment:
let's keep it simple, provide separately configuration for single node and cluster